storage/include/rocksdb/client.h

```diff
@@ -97,6 +97,16 @@ class Client : public concord::storage::IDBClient {
   concordUtils::Status multiPut(const SetOfKeyValuePairs& _keyValueMap) override;
   concordUtils::Status multiDel(const KeysVector& _keysVec) override;
   concordUtils::Status rangeDel(const Sliver& _beginKey, const Sliver& _endKey) override;
+
+  Status reclaimDiskSpace() override {
+    ::rocksdb::CompactRangeOptions opt;
+    opt.allow_write_stall = true;
+    opt.change_level = true;
+    auto s = dbInstance_->CompactRange(opt, nullptr, nullptr);
```

Comment:
I think this call will only compact the default column family: https://github.com/facebook/rocksdb/blob/master/include/rocksdb/db.h#L1143

I think we should call it per column family: https://github.com/facebook/rocksdb/blob/master/include/rocksdb/db.h#L1138

Reply:
fixed